src/base/path.h

```diff
@@ -40,9 +40,23 @@ class Path final
     Path() = default;
 
     explicit Path(const QString &pathStr);
-    explicit Path(const char pathStr[]);
     explicit Path(const std::string &pathStr);
 
+    template <std::size_t N>
+    explicit Path(const char (&pathStr)[N])
+        : Path(QString::fromLatin1(pathStr))
+    {
+        static_assert((strlen(pathStr) > 0), "String should not be empty");
```

Comment:
Empty path is allowed. Why do you forbid it?

Reply:
This is guarding for literals like `Path p("1234")`. Those shouldn't be empty.
However this commit is dropped due to technical difficulties.